anta/tests/snmp.py

```diff
@@ -237,3 +242,73 @@ def test(self) -> None:
             self.result.is_failure(f"Expected `{self.inputs.contact}` as the contact, but found `{contact}` instead.")
         else:
             self.result.is_success()
+
+
+class VerifySnmpPDUs(AntaTest):
```

Comment:
```suggestion
class VerifySnmpPDUCounters(AntaTest):
```

Reply:
Also change the docstrings, etc.

Reply:
Updated test name and docstring as well.